platforms/minestom/src/main/java/com/dfsek/terra/minestom/world/TerraMinestomWorldBuilder.java

```diff
@@ -0,0 +1,86 @@
+package com.dfsek.terra.minestom.world;
+
+import com.dfsek.terra.api.config.ConfigPack;
+
+import com.dfsek.terra.api.registry.CheckedRegistry;
+
+import com.dfsek.terra.minestom.MinestomPlatform;
+import com.dfsek.terra.minestom.api.BlockEntityFactory;
+import com.dfsek.terra.minestom.api.EntityFactory;
+import com.dfsek.terra.minestom.api.filter.ChunkFilter;
+import com.dfsek.terra.minestom.block.DefaultBlockEntityFactory;
+import com.dfsek.terra.minestom.entity.DefaultEntityFactory;
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.instance.Instance;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Random;
+import java.util.function.Function;
+
+
+public class TerraMinestomWorldBuilder {
```

Comment:
Why do we have this?

Reply:
The `TerraMinestomWorld` class has a lot of parameters. Since Minestom is not configured like bukkit, all worlds need to be created through the API. This makes it easier to create a world. See the provided example for more details